extras/travis_build.sh

```diff
@@ -108,7 +106,11 @@ if [ "$xLIBNEUROSIM" = "1" ] ; then
     CONFIGURE_LIBNEUROSIM="-Dwith-libneurosim=$HOME/.cache/libneurosim.install"
     chmod +x extras/install_csa-libneurosim.sh
     ./extras/install_csa-libneurosim.sh
-    export LD_LIBRARY_PATH=$HOME/.cache/csa.install/lib:$LD_LIBRARY_PATH
+    if [[ $OSTYPE == darwin* ]]; then
```

Comment:
Shouldn't this also rather use `uname` as above?

Reply:
Not necessary as travis_build only needs to run on Bash, not pure POSIX shells (/bin/sh). I could replace all the checks, but I'm afraid of feature creep and breaking stuff. However, if you feel it is neater and should be included in this PR I will add it.

Reply:
Thanks for the clarification. No need for changes from my side.